### PR TITLE
test(core): getHashedData tests added

### PR DESCRIPTION
**Pull request by Kuliak**

### CODE REVIEW

File: perun-core/src/test/java/cz/metacentrum/perun/core/entry/ServicesManagerEntryIntegrationTest.java [hunk 1]
```diff
@@ -1344,7 +1345,7 @@ public void addDestinationSameDestinationDifferentAdmin() throws Exception {
 	}
 
 	@Test
-	public void testGetHashedDataWithGroups() throws Exception {
+	public void getHashedDataWithGroups() throws Exception {
 		System.out.println(CLASS_NAME + "testGetHashedDataWithGroups");
```
Comment (Johaney-s) [inline perun-core/src/test/java/cz/metacentrum/perun/core/entry/ServicesManagerEntryIntegrationTest.java:1349]:
```suggestion
		System.out.println(CLASS_NAME + "getHashedDataWithGroups");
```

**Review by Johaney-s — COMMENTED** [review]
i'm not a fan of verifying the structure before setting the service to skip expired members, as it seems to me to duplicate the already existing basic tests. what do you think, @Kuliak, @mattjoke ?

**Comment by mattjoke** [comment]
> i'm not a fan of verifying the structure before setting the service to skip expired members, as it seems to me to duplicate the already existing basic tests. what do you think, @Kuliak, @mattjoke ?

Yes, personally I would skip the verification before as it is redundant.

**Review by Johaney-s — APPROVED** [review]

**Review by mattjoke — APPROVED** [review]
Looks good 👍

**Pull request merged**